bittensor/_endpoint/__init__.py

```diff
@@ -139,7 +139,7 @@ def from_tensor( tensor: torch.LongTensor) -> 'bittensor.Endpoint':
 
     @staticmethod
     def dummy():
-        return endpoint_impl.Endpoint(uid=-1,version=0, hotkey = "", ip_type = 4, ip = '0.0.0.0', port = 0, modality= 0, coldkey = "")
+        return endpoint_impl.Endpoint(uid=0, version=0, hotkey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", ip_type = 4, ip = '0.0.0.0', port = 0, modality= 0, coldkey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
```

Comment:
Same concern here, if dummy() is only being used in testing, let's put it in a test file.

Reply:
Not only in testing, we also fill these null objects when there is a parsing error from the chain.